dashboard/config/locales/scripts.en.yml

```diff
@@ -11515,3 +11515,11 @@ en:
               name: Final Project
               description_student: ''
               description_teacher: ''
+        csp1-pilot:
```

Comment:
Is this a script used for testing? Is this something that curriculum can use to start building? Why are you adding this script?

Reply:
Good question! This is a new script whose purpose is to prove that scripts with `pilot_experiment` work as we expect. once everyone is happy with how it works, the curriculum team could start using this script as part of the CSP pilot. it's not currently used for any automated testing. if we end up having UI tests for this feature, then I imagine we'd want to start using a dedicated test pilot script at that time so that tests won't break in the distant future when someone turns off pilot_experiment for this script.